src/pe/export.rs

```diff
@@ -173,16 +200,16 @@ impl<'a> Reexport<'a> {
 #[derive(Debug, Default)]
 /// An exported symbol in this binary, contains synthetic data (name offset, etc., are computed)
 pub struct Export<'a> {
-    pub name: &'a str,
+    pub name: Option<&'a str>,
     pub offset: usize,
     pub rva: usize,
-    pub size: usize,
+    // pub size: usize,
```

Comment:
Why was this field removed?

Reply:
Sorry, because I do not understand why this field is needed, I have restored `size` (but I still do understand :))